tasks/self_init.yml

```diff
@@ -0,0 +1,45 @@
+- name: self-init | Configure OpenBao (with orchestration)
+  ansible.builtin.template:
+    src: openbao.hcl.j2
+    dest: /etc/openbao/openbao.hcl
+    owner: root
+    group: root
+    mode: "0644"
+
+- name: self-init | Configure OpenBao environment variables for self-initialization
+  ansible.builtin.template:
+    src: openbao.env.j2
+    dest: /etc/openbao/openbao.env
+    owner: root
+    group: root
+    mode: "0644"
+  when:
+    - openbao_service_environment | length
+
+- name: self-init | Apply pending handler changes immediately (leader)
+  ansible.builtin.meta: flush_handlers
+  when:
+    - inventory_hostname == ansible_play_hosts[0]
+
+- name: self-init | Ensure OpenBao is enabled and started (leader)
+  ansible.builtin.systemd:
+    name: openbao
+    state: started
+    enabled: true
+    daemon_reload: true
```

Comment:
Don't think `daemon_reload` is required, as we're not changing the service file.